app/models/overview.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+# At-a-glance information for the POD aggregator and a single organization
+class Overview
+  attr_reader :organization
+
+  def initialize(user)
+    @organization = user.organization if user.present?
+  end
+
+  # total number of public organizations that provide data
+  def provider_count
+    providers.count
+  end
+
+  # most recent upload to default stream by any provider organization
+  def last_upload
+    Upload.includes(:organization, :stream)
```

Comment:
I'd be tempted to memoize these so the database calls only have to happen once per request.
```suggestion
    @last_upload ||=
      Upload.includes(:organization, :stream)
```